chainer/variable.py

```diff
@@ -508,12 +510,23 @@ class Variable(object):
     _grad = None
 
     def __init__(self, data=None, **kwargs):
-        # type: (types.NdArray, **tp.Any) -> None
+        # type: (tp.Optional[types.NdArray], **tp.Any) -> None
 
-        name, grad, requires_grad = argument.parse_kwargs(
-            kwargs, ('name', None), ('grad', None), ('requires_grad', True),
-            volatile='volatile argument is not supported anymore. '
-            'Use chainer.using_config')
+        # Retrieve keyword-only arguments as done in parse_kwargs()
+        # (These checks are required to keep backward compatibility)
+        name = kwargs.pop('name', None)
```

Comment:
Is this unrolling of kwargs handling still needed?

Reply:
I see. Fixed to use the original.